### ISSUE TITLE
Can't instantiate a TwilioRestClient with API Key and API Secret

### ISSUE THREAD

**Issue by acamino** [twilio/twilio-python#286]
**Version:** 5.6.0
**API Subdomain (api/taskrouter/ip_messaging):** API

There is no way to create a TwilioRestClient with an API Key and API Secret. https://github.com/twilio/twilio-python/blob/master/twilio/rest/base.py

This functionality is supported in other [Twilio Helper Libraries](https://www.twilio.com/docs/api/rest/keys#instance).

**Comment by johnpneumann** [comment]
Stumbled upon this issue as well, but found that there is a way to do it, but the documentation doesn't reflect that. Pull request #263 is out of date with master, but the doc fix would provide better guidance around how to do it. Providing an example here for anyone else who runs into it:

```
from twilio.rest import TwilioRestClient
api_key_sid = 'The api key sid from when you generated the token'
api_token = 'The actual token that you received'
account_sid = 'The main account sid'
client = TwilioRestClient(account=api_key_sid, token=api_token, request_account=account_sid)
```

**Comment by jingming** [comment]
This has been fixed in `6.0.0`!

**Comment by johnpneumann** [comment]
Thank you!
